recipe/meta.yaml

```diff
@@ -1,91 +1,158 @@
 {% set name = "napari" %}
-{% set version = "0.4.14" %}
+{% set version = "0.4.15rc2" %}
```

Comment:
Only merge when the final release is out. Otherwise, conda will see package as a new napari release (conda doesn't have the --pre flag as pypi).